src/Adyen/Service/Checkout.php

```diff
@@ -39,7 +39,7 @@ class Checkout extends \Adyen\ApiKeyAuthenticatedService
     /**
      * @var ResourceModel\Checkout\PaymentLinks
      */
-    protected $paymentLinks;
+    public $paymentLinks;
```

Comment:
Why is this public?

```suggestion
    protected $paymentLinks;
```

Reply:
Yeah so it's because this basically is the method that the merchant is calling, whereas for the other post methods it just functions as a private resource method.